### DISCUSSION TITLE
Multiple proxies

### DISCUSSION THREAD

**Discussion by claro-lukas** [MRColorR/money4band#23] [Q&A] [answered]
If I have multiple proxies to use, what is the best way to do it? Should I create different Docker container with each proxy written on .env file or is there easier way?

**Comment by MRColorR** [comment]
> If I have multiple proxies to use, what is the best way to do it? Should I create different Docker container with each proxy written on .env file or is there easier way?

To setup proxies you can use the auto setup runme script OR edit the env manually and then start the project using "docker compose up -d" . 

A single docker compose file can run all the apps containers simultaneously inside a single project.

If you want to run multiple projects instances on the same machine just run the runme again using different proxies or, if you want to keep track of all the different env files generated, copy the project in a different folder and run the setup script again. In both cases it will generate  several projects instances running in parallel

**Comment by claro-lukas** [comment]
Thank you for explanation.
One more question, if my proxies don't have username/password but instead are IP authenticated, how to use them in .env file (commented example contains only username:password proxies)?

**Reply by MRColorR** [reply, marked as answer]
Then your public IP address should be whitelisted/authenticated on your proxy. Regarding proxy configuration for the containers, just omit the proxy account and password parts in the proxy url, as they're optional parameters (it's just an example) and could be omitted if the proxy doesn't require them.
